src/test/java/com/fasterxml/jackson/core/format/DataFormatMatcherTest.java

```diff
@@ -0,0 +1,46 @@
+package com.fasterxml.jackson.core.format;
+
+import com.fasterxml.jackson.core.JsonFactory;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Unit tests for class {@link DataFormatMatcher}.
+ *
+ * @date 2017-07-31
+ * @see DataFormatMatcher
+ **/
+public class DataFormatMatcherTest {
+
+  @Test
+  public void testGetDataStream() throws IOException {
+    byte[] byteArray = new byte[2];
+    MatchStrength matchStrength = MatchStrength.WEAK_MATCH;
+    DataFormatMatcher dataFormatMatcher = new DataFormatMatcher(null,
+            byteArray,
+            0,
+            (-2),
+            null,
+            matchStrength);
+    InputStream inputStream = dataFormatMatcher.getDataStream();
+
+    assertEquals((-2), inputStream.available());
```

Comment:
Sure.

Considered it to be right returning a negative number.
However, read InputStream class comment of the JDK a minute ago and learned that it is not. right.